tests/e2e/entrypoint_secondary.sh

```diff
@@ -2,7 +2,8 @@
 # MONIKER=validator1|validator2|validator3|validator4 sh /opt/tests/e2e/entrypoint.sh
 
 # 1. chain init
-/usr/bin/xplad init $MONIKER --chain-id localtest_1-1 --home $XPLAHOME
+/usr/bin/xplad init $MONIKER --chain-id localtest_9001-1 --home $XPLAHOME
```

Comment:
9001 is a EVMOS chain id. I think rollback is better.
https://chainlist.org/?search=9001